aspnet/fundamentals/app-state/sample/src/AppState/Model/RequestEntry.cs

```diff
@@ -1,11 +1,7 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace AppState.Model
 {
-    [Serializable]
```

Comment:
@robbert229  you can search .net core types on this site http://packagesearch.azurewebsites.net/